cg/constants/report.py

```diff
@@ -207,14 +215,14 @@
 
 _REQUIRED_SAMPLE_METADATA_WTS_FIELDS: list[str] = _REQUIRED_SAMPLE_METADATA_SEQUENCING_FIELDS + [
     "bias_5_3",
+    "dv200",
     "input_amount",
     "mrna_bases",
     "pct_adapter",
     "pct_surviving",
     "q20_rate",
     "q30_rate",
     "ribosomal_bases",
-    "rin",
```

Comment:
we should keep the rin value. Both rin and dv200 should be shown

Reply:
Yes, they will. These are just the mandatory fields. If DIV200 is missing it will fail the report, but we're jusr allowing RIN N/A values here by removing it from that list

Reply:
oh, I don't think DV200 should be mandatory. Sorry if I wasn't clear. I meant we should show those values if we have them or NA if not. But they should not be mandatory

Reply:
I've removed it from mandatory fields 👍